integrations/authentication/OAuth2Manager.js

```diff
@@ -53,11 +55,13 @@ export default class {
 
   async authorize(openUrlAndWaitForCallback) {
     // Step 1 - user authorizes app
+    // "Automatic's" API doesn't accept the function's URL-encoded colons, hence scope attached separately
     const requestURLParams = objectToURLParams({
       client_id: this.clientId,
       redirect_uri: getCallbackUrl(),
       response_type: 'code',
-    });
+    }) + (this.scope || '');
```

Comment:
Can you move the `&` to OAuthManager? Otherwise, we need to remember to add the `&` every time we use the scope.
```suggestion
    }) + (this.scope ? `&${this.scope}` : '');
```

Reply:
Okay, changed it in the latest commit 👌